tb_plugin/torch_tb_profiler/profiler/trace.py

```diff
@@ -23,6 +23,8 @@ class EventTypes(object):
     MEMSET = "Memset"
     PYTHON = "Python"
     MEMORY = "Memory"
+    PYTHON_FUNCTION = "python_function"
```

Comment:
so whats the difference between `Python" on line 24 and our `python_function` ? the names could be very confusing to those new to this code

Reply:
the Python in 24 is used internally in tb_plugin. maybe we need some better name.